src/Surfnet/StepupGateway/GatewayBundle/Service/ResponseRenderingService.php

```diff
@@ -74,14 +74,14 @@ public function renderRequesterFailureResponse(ResponseContext $context)
      */
     public function renderUnprocessableResponse(ResponseContext $context)
     {
-        return $this->renderSamlResponse(
+        return ($this->renderSamlResponse(
             $context,
             'unprocessable_response',
             $this->responseBuilder
                 ->createNewResponse($context)
                 ->setResponseStatus(Constants::STATUS_RESPONDER)
                 ->get()
-        );
+        ));
```

Comment:
Wrapping the return statement in `(` brackets `)` is not adding any value right?